routing/routing_test.go

```diff
@@ -1,26 +1,18 @@
 package routing
 
 import (
+	"github.com/zalando/skipper/eskip"
 	"github.com/zalando/skipper/filters"
 	"github.com/zalando/skipper/filters/filtertest"
-	"github.com/zalando/skipper/requestmatch"
 	"github.com/zalando/skipper/routing/testdataclient"
 	"net/http"
 	"testing"
 	"time"
 )
 
-type matcher func(req *http.Request) (*Route, map[string]string)
+type matcherFunc func(req *http.Request) (*Route, map[string]string)
 
-func castMatcher(m *requestmatch.Matcher) matcher {
-	return func(req *http.Request) (*Route, map[string]string) {
-		v, p := m.Match(req)
-		r, _ := v.(*Route)
-		return r, p
-	}
-}
-
-func testMatcherWithPath(t *testing.T, m matcher, path string, matchRoute *Route) {
+func testMatcherWithPath(t *testing.T, m matcherFunc, path string, matchRoute *Route) {
 	req, err := http.NewRequest("GET", "http://www.example.com"+path, nil)
```

Comment:
all further ones use this one, too, because they use this function
in the test, it proxies requests, that were made to .com, to .org